app/uk/gov/hmrc/incometaxpenaltiesappealsfrontend/controllers/JointAppealController.scala

```diff
@@ -0,0 +1,88 @@
+
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.config.ErrorHandler
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers.predicates.{AuthAction, UserAnswersAction}
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.forms.JointAppealForm
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.models.PenaltyData
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.pages.JointAppealPage
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.services.UserAnswersService
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.utils.IncomeTaxSessionKeys
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.views.html._
+import uk.gov.hmrc.incometaxpenaltiesfrontend.controllers.predicates.NavBarRetrievalAction
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+
+class JointAppealController @Inject()(jointAppeal: JointAppealView,
+                                      val authorised: AuthAction,
+                                      withNavBar: NavBarRetrievalAction,
+                                      withAnswers: UserAnswersAction,
+                                      userAnswersService: UserAnswersService,
+                                      override val errorHandler: ErrorHandler,
+                                      override val controllerComponents: MessagesControllerComponents
+                                     )(implicit ec: ExecutionContext) extends BaseUserAnswersController {
+
+  def onPageLoad(): Action[AnyContent] = (authorised andThen withNavBar andThen withAnswers) { implicit user =>
+
+    user.userAnswers.getAnswerForKey[PenaltyData](IncomeTaxSessionKeys.penaltyData) match {
+      case Some(PenaltyData(_, _, _, Some(multiplePenaltiesData))) =>
+        Ok(jointAppeal(
+          form = fillForm(JointAppealForm.form(), JointAppealPage),
+          isAgent = user.isAgent,
+          firstPenaltyAmount = multiplePenaltiesData.firstPenaltyAmount.toString,
+          secondPenaltyAmount = multiplePenaltiesData.secondPenaltyAmount.toString
+        ))
+      case _ =>
+        NotImplemented
+      // TODO handle no multiple appeals data routing
+    }
+  }
+
+  def submit(): Action[AnyContent] = (authorised andThen withNavBar andThen withAnswers).async { implicit user =>
+
+    JointAppealForm.form().bindFromRequest().fold(
+      formWithErrors => {
+        user.userAnswers.getAnswerForKey[PenaltyData](IncomeTaxSessionKeys.penaltyData) match {
+          case Some(PenaltyData(_, _, _, Some(multiplePenaltiesData))) =>
+            Future.successful(BadRequest(jointAppeal(
+              form = formWithErrors,
+              isAgent = user.isAgent,
+              firstPenaltyAmount = multiplePenaltiesData.firstPenaltyAmount.toString,
+              secondPenaltyAmount = multiplePenaltiesData.secondPenaltyAmount.toString
+            )))
```

Comment:
It would be more TypeSafe to pass the `BigDecimal` value here through to the view. Then call a method within the view to render the amount in a currency format.